variables.tf

```diff
@@ -3,8 +3,8 @@ variable "namespace" {
   description = "Namespace for all resources, usually the organization or project name"
   type        = string
   validation {
-    condition     = length(var.namespace) <= 18 && can(regex("^[a-z0-9-]+$", var.namespace))
-    error_message = "Namespace must be lowercase alphanumeric and hyphens only, max 18 characters"
+    condition     = length(var.namespace) <= 12 && can(regex("^[a-z][a-z0-9-]+$", var.namespace))
+    error_message = "Namespace must be lowercase alphanumeric and hyphens only, start with a letter, max 12 characters"
```

Comment:
Actually, we might want to also add validation to the database_password variable as well.  I'll add a patch to this tomorrow morning.

> │ Error: creating RDS DB Instance (kay2-temp-db): operation error RDS: CreateDBInstance, https response error StatusCode: 400, RequestID: 3155baae-35d2-4623-8dab-17fbbd80626b, api error InvalidParameterValue: The parameter MasterUserPassword is not a valid password. Only printable ASCII characters besides '/', '@', '"', ' ' may be used.